Samples/excel-insert-file/README.md

```diff
@@ -58,14 +58,14 @@ To run the sample you just need to sideload the manifest. The add-in web files a
 1. Download the **manifest.xml** and **SalesTemplate.xlsx** files from this sample to a folder on your computer.
 1. Open [Office on the web](https://www.office.com/).
 1. Choose **Excel**, and then open a new document.
-1. Select the **Insert** tab, and choose **Office Add-ins**.
+1. Select the **Insert** tab, and choose **Add-ins**.
```

Comment:
I don't have Add-ins on the Insert tab anymore. Just the Home tab. Not sure if that's the default or a flighting thing.

Reply:
Probably a flighting thing.